shared/src/utilities/createLogger.js

```diff
@@ -0,0 +1,43 @@
+const combine = require('logform/combine');
+const json = require('logform/json');
+const printf = require('logform/printf');
+const timestamp = require('logform/timestamp');
+const {
+  createLogger: createWinstonLogger,
+  format,
+  transports,
+} = require('winston');
+const { cloneDeep, unset } = require('lodash');
+
+const redact = format(logEntry => {
+  const copy = cloneDeep(logEntry);
+  ['token', 'headers.authorization'].forEach(k => unset(copy, k));
```

Comment:
This redacts `token` and `headers.authorization` from any logged object, accomplishing https://github.com/flexion/ef-cms/issues/6408 for in-application logging. 

https://github.com/flexion/ef-cms/issues/6408 should still address any logging happening outside of the application.

Reply:
That seems apropos - the any potentially sensitive output from shell scripts and Honeybadger being the low-hanging fruit.